ui/src/app/shared/genericComponents/chart/chart.ts

```diff
@@ -37,9 +37,12 @@ export class ChartComponent implements OnInit, OnChanges {
     this.service.setCurrentComponent('', this.route).then(edge => {
       this.edge = edge;
     });
+
   }
 
+  /** Run change detection explicitly after the change, to avoid expression changed after it was checked*/
   ngOnChanges() {
```

Comment:
See https://angular.io/guide/styleguide#style-09-01

Reply:
ng lint is only throwing warning right now, probably should change this to error